Code/JavaWrappers/FilterCatalog.i

```diff
@@ -163,7 +163,7 @@
 %typemap(javaout) std::string RDKit::FilterCatalog::Serialize {
   return $jnicall;
 }
-%typemap(out) std::string RDKit::FilterCatalog::Serialize {
+%typemap(javaout) std::string RDKit::FilterCatalog::Serialize {
```

Comment:
This should indeed be %typemap(out), it needs to return a jbytearray and not a javastring